matter_server/client/model/node.py

```diff
@@ -16,6 +17,7 @@ class MatterNode:
     """Matter node."""
 
     root_device: MatterDevice[device_types.RootNode]
+    node_lock = asyncio.Lock()
```

Comment:
You need to initialize it in the constructor to make it unique per Node.

Reply:
One that that is meh here is that this ties the lock to the agnostic model. I wonder if instead we should store this on the HA side?